README.md

```diff
@@ -99,15 +86,7 @@ The plugin includes a Recommendations Block that uses the [Parse.ly API](https:/
    ![The main settings screen of the wp-parsely plugin](.wordpress-org/screenshot-2.png)
 3. Parse.ly plugin advanced settings. To be used only if instructed by Parse.ly staff.  
    ![The main settings screen of the wp-parsely plugin](.wordpress-org/screenshot-3.png)
-4. The settings for the Parse.ly Recommended Widget. Engage your visitors with predictive and personalized recommendations from Parse.ly.  
+4. The settings for the legacy Parse.ly Recommended Widget. Engage your visitors with predictive and personalized recommendations from Parse.ly.  
    ![The settings for the Parse.ly Recommended Widget](.wordpress-org/screenshot-4.png)
 5. A view of the Parse.ly Dashboard Overview. Parse.ly offers analytics that empowers you to better understand how your content is peforming.  
    ![The Parsely Dashboard Overview](.wordpress-org/screenshot-5.png)
-
-## Frequently Asked Questions
-
-See [frequently asked questions](https://www.parse.ly/help/integration/wordpress#frequently-asked-questions) on the Parse.ly Technical Documentation.
-
-## Changelog
```

Comment:
FAQ and Changelog are defined / understood headings for WPORG page - was the intent to remove these completely?

Reply:
When someone (not on VIP), can view the modal for a plugin update, they may be checking what the changelog is - removing the section, along with a link to the changelog.md, makes it harder for them. 

No objection to removing the FAQ.

Reply:
Thanks for bringing this to my attention, as I didn't know that those sections were creating tabs in the plugin update/install modal.

I do see the utility of having this for people who use those tabs, so I've restored the sections with revised texts that point to the appropriate resources.